x-pack/platform/plugins/shared/lens/public/persistence/lens_client.ts

```diff
@@ -37,10 +46,14 @@ export class LensClient {
   }
 
   async create(
-    { description, visualizationType, state, title, version }: LensAttributes,
+    { description, visualizationType, state, title, version }: LooseLensAttributes,
     references: Reference[],
     options: LensCreateRequestBody['options'] = {}
   ) {
+    if (visualizationType === null) {
+      throw new Error('Missing visualization type');
+    }
```

Comment:
Allow this to align with outside types from `LensDocument` for now and throw if trying to save a `null` viz type.